specification/hybridcompute/resource-manager/Microsoft.HybridCompute/preview/2022-08-11-preview/examples/Extension_List.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
Rename the example file to - ExtensionMetadata_Get

Reply:
Updated